syslog-ng/Dockerfile

```diff
@@ -20,4 +20,9 @@ EXPOSE 514/udp
 EXPOSE 601/tcp
 EXPOSE 6514/tcp
 
+HEALTHCHECK --interval=2m --timeout=3s --start-period=30s CMD syslog-ng-ctl stats || exit 1
```

Comment:
[optional] I would use the full path for `syslog-ng-ctl`, it is safer imho. (even if docker documentation states it is okay)

Reply:
[optional] If you only care about that "syslog-ng will answer or not on it's control channel", than `show-license-info` uses much less resource than the stats command. (Stats also highly depend on the current configuration.)

Otherwise LGTM.

Reply:
@drexlma could you fix both above? Me seems both are nice improvement, and easy to fix. Full path and show-license-info? Also, could you remove the unnecessarily added new lines below? Otherwise, this can be merged.